OneCommunityGlobal/HighestGoodNetworkApp
# Haohui - fix permission for edit tasks

# Description
<img width="691" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/78375725/90f45958-f996-4781-a325-308e294ece37">


## How to test:
1. check into current branch
2. do npm run build and npm run start to run this PR locally

**Permissions assign to individuals**

3. log as owner user call user A
4. go to dashboard→ Other Links→ Permissions Management→ Manage User Permissions
5. input another user name, call user B
6. find the “Edit Task”, click the add button
7. scroll down and click submit button
8. log out and login as user B(if already logged in, make sure to log out and log back in)
9. go to “Dashboard”→ "Other Links" -> "Projects" -> "WBS Button" -> "Choose any WBS" -> "Edit" -> "Edit" 
10. make some change of the task then submit.

**Managers account**

11. log as owner user call user A
12. go to dashboard→ Other Links→ Permissions Management→ Manage User Permissions
13. input manager user name, call user C
14. click the "Reset to Default" button on top right
15. scroll down and click submit button (double check "Edit Task" is unselected)
16. log out and login as user C(if already logged in, make sure to log out and log back in)
17. go to “Dashboard”→ "Other Links" -> "Projects" -> "WBS Button" -> "Choose any WBS" -> "Edit" -> "Edit" 
18. make some change of the task then submit.

## Screenshots or videos of changes:
Managers account

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/78375725/c8d6239e-9d8f-486d-9ab6-b186984c3359


Permissions assign to individuals



https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/78375725/c47f1ddd-275d-4452-b8d8-fd676b236d45




## Note:
Include the information the reviewers need to know.
